conf/iptables.conf.example

```diff
@@ -78,6 +78,8 @@
 -A input-internal-vlan-if --protocol tcp --match tcp --dport 80  --jump ACCEPT
 -A input-internal-vlan-if --protocol tcp --match tcp --dport 443 --jump ACCEPT
 -A input-internal-vlan-if --protocol tcp --match tcp --dport 647 --jump ACCEPT
+# HTTP (parking portal)
+-A input-internal-vlan-if --protocol tcp --match tcp --dport 5252 --jump ACCEPT
```

Comment:
It can be a constant but it won't be usable in this file unless we inject it in the iptables manager.

That's why I put it this way.

Do you think it would be better to inject it in iptables.pm when we generate ?